massa-api/src/public.rs

```diff
@@ -212,11 +214,15 @@ impl Endpoints for API<Public> {
                         address: caller_address,
                         coins: Default::default(),
                         owned_addresses: vec![caller_address],
+                        //TODO: Get this from the api parameters
```

Comment:
this one is always None

Reply:
In a read only call we don't pass op datastore it's just on execute ?

Reply:
For now we don't pass it on function calls (eg. callSC) nor readonly func calls.

However it would be interesting to add it to ReadOnlyExecutionTarget::BytecodeExecution